### ISSUE TITLE
LWC Jest tests failing when adding variant to lightning-radio-group

### ISSUE THREAD

**Issue by srgclr** [salesforce/sfdx-lwc-jest#73]
We have developed a custom LWC that uses the lightning-radio-group component and sets the variant so we can hide the label (keeping the label there for accessibility purposes). This works fine on our scratch orgs. However, this causes our jest tests to fail with:

`Error: Unknown public property "variant" of element <lightning-radio-group>`

Removing the variant property allows the tests for our component to pass, but this won't give us the UX we require.

We are using the following dependencies to pull in the LWC jest test framework with npm:

```
"devDependencies": {
  "@salesforce/lwc-jest": "^0.5.1",
  "js-yaml": "3.13.1",
  "jest": ">= 23.0.0"
}
```
Is there a way to get lwc-jest to work with the newest versions of the standard LWCs?
